build-tools/xaprepare/xaprepare/Scenarios/Scenario_AndroidTestDependencies.cs

```diff
@@ -15,6 +15,7 @@ protected override void AddSteps (Context context)
 			Steps.Add (new Step_InstallAdoptOpenJDK8 ());
 			Steps.Add (new Step_InstallMicrosoftOpenJDK11 ());
 			Steps.Add (new Step_Android_SDK_NDK (AndroidToolchainComponentType.CoreDependency));
+			Steps.Add (new Step_GenerateFiles (atBuildStart: false));
```

Comment:
@jonpryor yes, I found for this to work, we need to generate `bin\Build$(Configuration)\Mono.Android.Apis.projitems` on test lanes...

Having trouble figuring out what's wrong here...